extras/create_send_notifications.py

```diff
@@ -188,6 +201,11 @@ def main():
     else:
         logging.info("Nothing to email - skipping this step")
 
+    # Remove orgs from notifications_to_delete if they are in the list of orgs
+    # that we just generated notifications for (most likely because the
+    # notification was included in an ancestor org's notification)
+    notifications_not_generated = sorted(set(notifications_to_delete) - set(orgs_notified))
```

Comment:
Apologies that this was unclear in https://github.com/cisagov/cyhy-reports/pull/104#discussion_r1416154995:
```suggestion
    notifications_to_delete = sorted(set(notifications_not_generated) - set(orgs_notified))
```

and then `notifications_not_generated` instead of `notifications_to_delete` in
https://github.com/cisagov/cyhy-reports/blob/2e1377549d690b7d7899661bf9e01ea4eb2d2d54/extras/create_send_notifications.py#L141